pl_bolts/models/self_supervised/cpc/cpc_module.py

```diff
@@ -39,7 +39,7 @@ class CPCV2(pl.LightningModule):
 
     def __init__(
             self,
-            datamodule: pl_bolts.datamodules.LightningDataModule = None,
+            datamodule: pl.LightningDataModule = None,
```

Comment:
I would prefer direct import not with `pl.`

Reply:
i’d prefer pl.

get people in the habit of doing like numpy